README.md

```diff
@@ -150,3 +150,99 @@ Currently in Doppler [we are always using the v1](https://github.com/MakingSense
   </body>
 </html>
 ```
+## File storage
+
+### AWS S3 storage configuration
+
+Unlayer have file storage support built-in with Amazon S3 that can be configured without any coding. Follow these steps to set up your S3 file storage.
+
+- Create S3 Bucket: The first step is to create a bucket in your AWS S3 console. You need to uncheck "'Block all public access" for S3 storage.
+
+![AWS S3 edit block puplic access](./docs/aws-uncheck-s3-block-all-public-access.png)
+- Create IAM User: Go to your AWS IAM console and create a new user with `Programmatic Access` so you can get the access key id and secret access key.
+- Attach Policy and Permissions: Aattach a minimum set of permissions, use the following policy.
+Make sure to replace BUCKET-NAME, ID-COUNT, ID-USER with your actual bucket settings.
+```json
```

Comment:
Interlines and spaces to make the image part of the first item:
```suggestion
- Create S3 Bucket: The first step is to create a bucket in your AWS S3 console. You need to uncheck "'Block all public access" for S3 storage.

  ![AWS S3 edit block puplic access](./docs/aws-uncheck-s3-block-all-public-access.png)

- Create IAM User: Go to your AWS IAM console and create a new user with `Programmatic Access` so you can get the access key id and secret access key.

- Attach Policy and Permissions: Aattach a minimum set of permissions, use the following policy.

Make sure to replace BUCKET-NAME, ID-COUNT, ID-USER with your actual bucket settings.

```json
```

And, if _"Make sure..."_ is subordinate to the third item, you should add 2 spaces before, the same for the json example.

Reply:
Changed.